LICENSES/3rdpartylicenses.md

```diff
@@ -0,0 +1,20 @@
+## The used third party dependencies and their licenses
+
+| Dependency                                         | License      |
+| -------------------------------------------------- | ------------ |
+| github.com/BurntSushi/toml                         | MIT          |
+| github.com/PaesslerAG/jsonpath                     | BSD-3-Clause |
+| github.com/PaesslerAG/gval                         | BSD-3-Clause |
+| github.com/ProtonMail/gopenpgp/v2                  | MIT          |
+| github.com/jessevdk/go-flags                       | BSD-3-Clause |
+| github.com/mitchellh/go-homedir                    | MIT          |
+| github.com/santhosh-tekuri/jsonschema              | Apache-2.0   |
+| golang.org/x/crypto                                | BSD-3-Clause |
+| github.com/ProtonMail/go-crypto                    | BSD-3-Clause |
+| github.com/ProtonMail/go-mime                      | MIT          |
+| github.com/konsorten/go-windows-terminal-sequences | MIT          |
+| github.com/pkg/errors                              | BSD-2-Clause |
```

Comment:
Unrelated to the license info, just triggered by sampling some repositories on the correctness:
> With the upcoming Go2 error proposals this package is moving into maintenance mode. The roadmap for a 1.0 release is as follows

It is no problem that `github.com/pkg/errors` has been archived, right?